File: lesson6/python3/homework/homework6_7.py
#!/usr/bin/python3
#: lesson6/python3/homework/homework6_7.py
# Напишите функции, которая вычисляет факториал натурального числа N.

def factorial(numb_n):
    res = 1
    for i in range(1, numb_n + 1):
        res *= i
    return res

numb = 5
result = factorial(numb)
print(result)

File: lesson6/c_c++/homework/homework6_7.cpp
/*
 * lesson6/c_c++/homework/homework6_7.cpp
 */
#include <iostream>

int factorial(int numb_n) {
    int res = 1;
    for (int i = 1; i <= numb_n; i++) {
        res *= i;
    }
    return res;
}

int main(int argc, char* argv[]) {
    int numb = 5;
    int result = factorial(numb);
    std::cout << result << '\n';
    return 0;
}
